gulpfile.js/tasks/html.js

```diff
@@ -37,7 +37,7 @@ var htmlTask = function() {
     .on('error', handleErrors)
     .pipe(gulpif(global.production, htmlmin(config.tasks.html.htmlmin)))
     .pipe(gulp.dest(paths.dest))
-    .pipe(browserSync.stream())
+    .on('end', browserSync.reload)
```

Comment:
It sounds like that should work based on the docs, but I haven't noticed any difference when trying that method.  It doesn't seem to really matter until you have a lot of files or a longer nunjucks processing time, but once either of those are an issue using `.on('end', browserSync.reload)` is the only thing I've seen that consistently solves the problem.  Discussed it a little bit in #251, and it didn't sound like anyone else had luck with `stream: true` either.